docs-website/sidebars.js

```diff
@@ -113,6 +113,11 @@ module.exports = {
     "Metadata Modeling": [
       "docs/modeling/metadata-model",
       "docs/modeling/extending-the-metadata-model",
+      {
+        label: "Metadata Events",
+        type: "doc",
+        id: "docs/what/mxe",
```

Comment:
Hi there! In a recent version of datahub, we added 2 new important events that are meant to replace the MAE / MCE events. This is discussed in this doc: https://datahubproject.io/docs/advanced/mcp-mcl/

We should update /what/mxe to point to this new page. In the short term, maybe we could have 2 entries in this sidebar: 

1. MetadataChangeEvent & MetadataAuditEvent -- points to docs/what/mxe
2. MetadataChangeProposal & MetadataChangeLog -- points to the link above

What do you think?

Thanks for the contribution!

Reply:
Adding links to both the Metadata Event pages in the nav bar wouldn't pose any problems to me personally because I'm already familiar with Datahub by now. But for the sake of newcomers I'd recommend having just one link, or no link at all - compared to two links. See rest of my comment for the justification.

I feel that having two lop level links for Metadata Events will be confusing for newcomers. Which brings me to a larger point: The story around the MCE -> MCP transition is not at all clear from the docs.  The MAE page (https://datahubproject.io/docs/what/mxe) has no mention of the newer events. Even I didn't know about MCP for quite a while after finding this page.

The MCP page (https://datahubproject.io/docs/advanced/mcp-mcl/) starts off with a huge wall of text and is hard to understand. For example, this sentence in the 2nd para is impenetrable to me: "This provides flexibility, allowing for the core models comprising the Metadata Graph to be added and changed dynamically, without requiring structural updates to Kafka or REST API schemas used for ingesting and serving Metadata."

I'd like to see the MCP page improved, and a new page for Metadata Events pointing to what's the recommended way today, what's the legacy way etc. Until then I'm happy to leave both of these off the nav bar. What do you think?

Reply:
Yeah I'm totally with you on this one. Agree we need another pass at these before we put them front and center. Is this something you'd be interested in working on? If not I will loop back with the team with this feedback and get back to you on the timeline

Reply:
You're right. I won't take this up because I don't have the context. I have reverted the change to the sidebar.js file.

The PR is ready for approval now.